sagemaker-debugger/tensorflow2/tensorflow2_keras_to_estimator/tf2-keras-model-to-estimator.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>Since our demo training script <code>tf_keras_to_estimator.py</code> uses the <code>tensorflow-datasets</code> package that is not available in the pre-built container, we need to install it as a requirement for the training job. To add the installation step, we simply add the package in a <code>requirement.txt</code> file. You can add a list of required libraries to run your own training script in the same way. </p><p> </p><p>The <code>launcher.sh</code> script is provided along with this notebook, which will install the <code>tensorflow-datasets</code>package and initiate the training script. For more information about installing third-party libraries, see <a href="https://sagemaker.readthedocs.io/en/stable/frameworks/tensorflow/using_tf.html#use-third-party-libraries" target="_blank">Use third-party libraries with SageMaker Python SDK</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done in latest</p>